import "session_commands.proto";

message Command_DeckDel {
	extend SessionCommand {
		optional Command_DeckDel ext = 1011;
	}
	optional uint32 deck_id = 1;
}
